cloud-controller-manager/do/loadbalancers.go

```diff
@@ -476,6 +482,17 @@ func getProtocol(service *v1.Service) (string, error) {
 	return protocol, nil
 }
 
+// getHealthCheckPath returns the desired path for health checking
+// health check path should default to / if not specified
+func getHealthCheckPath(service *v1.Service) string {
+	path, ok := service.Annotations[annDOHealthCheckPath]
+	if !ok {
+		return ""
```

Comment:
Shouldn't this `return "/"`?

Reply:
I think godo passes in ""  but have to double check.

Reply:
godo passes in `""` for the request, meaning "let server side decide default" which I think is the correct behavior here.

Reply:
The docs for the annotation says the default is `/`, though. Is the godoc there wrong, or this code path wrong?